core/src/main/java/org/infinispan/xsite/events/XSiteEventType.java

```diff
@@ -0,0 +1,17 @@
+package org.infinispan.xsite.events;
+
+/**
+ * Types of {@link XSiteEvent}.
+ *
+ * @since 15.0
+ */
+public enum XSiteEventType {
+   /**
+    * Sent from site A to site B, notifies site B that a new connection to A is available.
+    */
+   SITE_CONNECTED,
+   /**
+    * Sent from site A to site B, notifies site B that it can send state (automatic cross-site state transfer must be enabled).
+    */
+   REQUEST_STATE
```

Comment:
Massive nitpick, feel free to ignore :slightly_smiling_face:  . `/s/REQUEST_STATE/STATE_REQUEST`.

Reply:
I'll change :+1: